solidity-v1/dashboard/src/services/threshold-authorization.service.js

```diff
@@ -0,0 +1,90 @@
+import {
+  ContractsLoaded,
+  getContractDeploymentBlockNumber,
+  getThresholdTokenStakingAddress,
+} from "../contracts"
+import { getOperatorsOfAuthorizer } from "./token-staking.service"
+import {
+  AUTH_CONTRACTS_LABEL,
+  TOKEN_STAKING_CONTRACT_NAME,
+} from "../constants/constants"
+import { Keep } from "../contracts"
+import { isSameEthAddress } from "../utils/general.utils"
+
+const fetchThresholdAuthorizationData = async (address) => {
+  if (!address) {
+    return []
+  }
+  const thresholdTokenStakingContractAddress = getThresholdTokenStakingAddress()
+  const { stakingContract } = await ContractsLoaded
+  const authorizerOperators = await getOperatorsOfAuthorizer(address)
+  const authorizationData = []
+
+  const keepToTStakedEvents =
+    await Keep.keepToTStaking.getStakedEventsByOperator(address)
+
+  const operatorsStaked = keepToTStakedEvents.map(
+    (event) => event.returnValues.stakingProvider
+  )
+
+  const stakesParticipants = (
+    await stakingContract.getPastEvents("OperatorStaked", {
+      fromBlock: await getContractDeploymentBlockNumber(
+        TOKEN_STAKING_CONTRACT_NAME
+      ),
+      filter: { operator: authorizerOperators },
+    })
+  ).map((event) => {
+    return {
+      authorizer: event.returnValues.authorizer,
+      operator: event.returnValues.operator,
+      beneficiary: event.returnValues.beneficiary,
+    }
+  })
```

Comment:
We can get this values if we filter `OperatorStaked` event from `KeepTokenStaking` contract by `authorizer` so maybe we should refactor `getOperatorsOfAuthorizer` fn eg:
`keep-core/solidity-v1/dashboard/src/services/token-staking.service.js`
```
export const getAllOperatorStakedEventsByAuthorizer = async (authorizer) => {
 const { stakingContract } = await ContractsLoaded
  return 
    await stakingContract.getPastEvents("OperatorStaked", {
      fromBlock: await getContractDeploymentBlockNumber(
        TOKEN_STAKING_CONTRACT_NAME
      ),
      filter: { authorizer },
    }
}

export const getOperatorsOfAuthorizer = async (authorizer) => {
  return (
    await getAllOperatorStakedEventsByAuthorizer(authorizer
  ).map((_) => _.returnValues.operator)
}
```

and then in `threshold-authorization.service.js` we will use `getAllOperatorStakedEventsByAuthorizer` and we will get `authorizer, operator, beneficiary`  w/o additional call to a node.